src/pages/docs/getting-started/index.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Overview"
+meta_description: "Get started with Pub/Sub in various languages or frameworks using Ably. Learn how to publish, subscribe, track presence, fetch message history, and manage realtime connections."
+meta_keywords: "Pub/Sub, Ably SDKs, realtime messaging, publish subscribe, getting started guides, realtime communication, Ably tutorial, message history, presence API, Ably CLI Pub/Sub"
+---
+
+Ably is a realtime experience infrastructure provider that powers synchronized digital experiences in realtime. Our platform provides APIs and SDKs to help developers build realtime features like live chat, data broadcast, notifications, and collaborative experiences. With Ably's Pub/Sub messaging, you can publish messages to channels and have them delivered instantly to millions of subscribers across any device, anywhere in the world.
+
+Choose from the following list of getting started guides to build an application using the Pub/Sub product. Each guide is tailored for different programming languages or frameworks and provides step-by-step instructions and code examples to help you integrate Pub/Sub into your application.
+
+<Tiles>
+{[
+  {
+    title: 'JavaScript',
+    description: 'Start building with Pub/Sub using our JavaScript SDK',
```

Comment:
No 'our' please 🙂 

Maybe something like 'Learn the basics of building with the Ably Pub/Sub SDK in JavaScript'?